### PR TITLE
rbt: add page

### PR DESCRIPTION
**Pull request by tmoon8730**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

As requested by issue #2782

### CODE REVIEW

File: pages/common/rbt.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# rbt
+
+> RBTools is a set of command line tools for working with Review Board and RBCommons.
+> Homepage: <https://www.reviewboard.org/docs/rbtools/dev/>.
+
+- Post changes to Review Board:
```
Comment (sbrl) [inline pages/common/rbt.md]: Should this be ....`to a review board`? Or is `review board` the name of the software itself?

**Comment by tmoon8730** [comment]
I believe thats the software itself

File: pages/common/rbt.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display the diff that will be sent to Review Board:
+
+`rbt diff`
+
+- Land a change in a local branch or on a review request:
+
+`rbt land {{branchname}}`
+
+- Patche your tree with a change on a review request:
```
Comment (mebeim) [inline pages/common/rbt.md]:
```suggestion
- Patch your tree with a change on a review request:
```

Typo.

File: pages/common/rbt.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# rbt
+
+> RBTools is a set of command line tools for working with Review Board and RBCommons.
+> Homepage: <https://www.reviewboard.org/docs/rbtools/dev/>.
+
+- Post changes to Review Board:
+
+`rbt post {{changenum}}`
```
Comment (mebeim) [inline pages/common/rbt.md]:
```suggestion
`rbt post {{change_number}}`
```
Clearer token name.

File: pages/common/rbt.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display the diff that will be sent to Review Board:
+
+`rbt diff`
+
+- Land a change in a local branch or on a review request:
+
+`rbt land {{branchname}}`
```
Comment (mebeim) [inline pages/common/rbt.md]:
```suggestion
`rbt land {{branch_name}}`
```
Clearer token name.

File: pages/common/rbt.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Patche your tree with a change on a review request:
+
+`rbt patch {{reviewrequestid}}`
```
Comment (mebeim) [inline pages/common/rbt.md]:
```suggestion
`rbt patch {{review_request_id}}`
```

Clearer token name.

File: pages/common/rbt.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set up RBTool to talk to your repository:
```
Comment (agnivade) [inline pages/common/rbt.md]: Minor nit: we try to avoid having direct speech in our pages. Please replace your with one's or anything else you feel like.

Comment (mebeim) [reply pages/common/rbt.md]: I'd suggest:

> Set up RBTool to talk to a repository:

File: pages/common/rbt.md (hunk 2 above)
Comment (tmoon8730) [reply pages/common/rbt.md]: Oops. Fixed

File: pages/common/rbt.md (hunk 3 above)
Comment (tmoon8730) [reply pages/common/rbt.md]: Makes sense. Fixed.

File: pages/common/rbt.md (hunk 4 above)
Comment (tmoon8730) [reply pages/common/rbt.md]: Fixed

File: pages/common/rbt.md (hunk 5 above)
Comment (tmoon8730) [reply pages/common/rbt.md]: Fixed

File: pages/common/rbt.md (hunk 6 above)
Comment (tmoon8730) [reply pages/common/rbt.md]: Changed

**Pull request merged**